Library/ViewModels/SearchViewModel.swift

```diff
@@ -185,10 +186,27 @@ public final class SearchViewModel: SearchViewModelType, SearchViewModelInputs,
 
     self.scrollToProjectRow = self.transitionedToProjectRowAndTotalProperty.signal.skipNil().map(first)
 
-    // KSRAnalytics
+    // Tracking
 
-    viewWillAppearNotAnimated
-      .observeValues { AppEnvironment.current.ksrAnalytics.trackProjectSearchView() }
+    let projectsProperty = Property(
+      initial: [],
```

Comment:
You probably want to try something closer to `self.projects.signal.combineLatest`. Something like that. There should be a relevant operator.